pkg/util/command_util.go

```diff
@@ -389,13 +389,13 @@ func GetUserFromUsername(userStr string, groupStr string, fallbackToUID bool) (s
 }
 
 func Lookup(userStr string) (*user.User, error) {
-	userObj, err := user.Lookup(userStr)
+	userObj, err := LookupUser(userStr)
```

Comment:
@greut would you mind clarifying why you chose not to use `os.User.Lookup`?

Reply:
I've got lost while searching for a viable solution. `os/user.Lookup` is more than good enough.